Dockerfile

```diff
@@ -28,16 +29,40 @@ FROM base as builder
 RUN apk --update --no-cache add $BUILD_DEPS
 
 # create venv
-RUN pip install -U pip pipenv
+RUN pip install -U pip wheel
+
+# pipenv install
+# RUN pip install -U pipenv
+
+# poetry install
+RUN pip install "poetry==$POETRY_VERSION"
 RUN python3 -m venv "$VIRTUAL_ENV"
 
 WORKDIR /build
+
+# PipFile install
+# COPY Pipfile* ./
+# RUN pipenv install --dev --ignore-pipfile
+
 # individual files here instead of COPY . . for caching
-COPY Pipfile* ./
-COPY Ctl/VERSION Ctl/
-RUN pipenv install --dev --ignore-pipfile
+COPY pyproject.toml poetry.lock ./
+
+RUN . "$VIRTUAL_ENV"/bin/activate && poetry install --no-root
```

Comment:
@egfrank we want `--no-dev` here as well

Not sure we need to activate here, although it doesn't hurt much either

Reply:
I had thought we needed to activate so that poetry knew to install into that virtualenv instead of creating its own, but I think I'm wrong.

Reply:
Fixed in https://github.com/fullctl/ixctl/pull/38/commits/45e8c730a4f2ec93acc17b2d7f4f5b3156174949